### PR TITLE
Updated test script for status

### PR DESCRIPTION
**Pull request by ananashraf**
- State change is being considered.

Instructions :
- Go to your local repository of KOBman and run the following command from tests/commands
    ./test-kob-status dummyenv 0.0.3 ( other versions are 0.0.2 ,0.0.5)

Output : 
----------

Screenshot from 2020-07-07 12-50-27

Failing Condition :
- Run the following command : ./test-kob-status dummyenv 0.0.1
image

### CODE REVIEW

**Comment by ananashraf** [comment]
Please ignore the kobman-help.sh file.

**Comment by asa1997** [comment]
LGTM

File: tests/commands/test-kob-status.sh [hunk 1]
```diff
@@ -1,106 +1,173 @@
...
-        __kobman_echo_no_colour "Please install an enivornment first and try again"
-        __kobman_echo_no_colour "Exiting!!!"
-        status="false"
-        return 1
-    else
-        __kobman_echo_no_colour "Environments found"
-        __kobman_echo_no_colour "Proceeding with the test..."
     fi
 
+    export INTERACTIVE_MODE="false"
```
Comment (panickervinod) [inline tests/commands/test-kob-status.sh:23]: Please rename INTERACTIVE_MODE to KOBMAN_INTERACTIVE_USER_MODE, all KOBman export variables will have KOBMAN_ prefix.

**Comment by asa1997** [comment]
@ananashraf Please close this pull request and raise a new one with the requested changes

**Review by asa1997 — CHANGES_REQUESTED** [review]
@ananashraf  Please close this pull request and raise a new one with the said changes.

**Pull request closed without merging**